server/src/concepts/offer/useCases/getOffers.js

```diff
@@ -0,0 +1,7 @@
+// not sure if i need to create command for getOffers
+// import {  } from "../repositories/commands.js"
+
+// this query is ready
+// import { getAllOffers } from "../repositories/queries.js";
```

Comment:
please provide getOffers function here in useCases and we can later expand it with options